### PR TITLE
Editing errors for consistency

### PR DESCRIPTION
**Pull request by hdauvic**
This PR solves #176 , the specified line codes seem to have been a legacy issue and long out of date. The hard coded line numbers have been removed, and the error is printed to the debugging console - it will be up to future developers to trace the issue themselves if that error is ever thrown. 

This PR also solves #4 as I found the culprits for the repeated printing of the unnecessary log and have commented them out with an additional comment above them if future devs want to use those for debugging.

I also commented out the repetitive `url of repo` log, this console line was overwhelming at times if the user has copious amounts of repos associated with their username. Again, a comment was added to inform future dev's of this choice. While this solves the pictured issue in #20 it does **NOT** resolve that issue - with all the console logs of the graph, that feature is still highly recommended.

Finally, I combed through the console messages to update them for grammatical consistency, punctuation, clarity, and standardized error printing to include a descriptive error comment before printing out the error log. Each error now reads `ERROR` in capital letters for easier viewing in the debugging console when an error is thrown.

When reviewing this PR please ensure that none of the functionality that you've implemented has been broken by the addition of console logs. I doubt it would be, but I just want to make sure.

### CODE REVIEW

File: app/misc/git.ts [hunk 1]
```diff
@@ -39,7 +39,7 @@ function sortedListOfCommits(commits){
 
     while (commits.length > 0) {
       let commit = commits.shift();
-      let parents = commit.parents();
+      let parents = commit.parentxs();
```
Comment (Athenasko) [inline app/misc/git.ts]: Hi @hdauvic, would you mind elaborating on the change here? Was the addition of the 'x' a typo or is it a call in typescript? I couldn't find any reference and definitions other than here so I thought I'd ask for clarification. Thanks!

Comment (hdauvic) [reply app/misc/git.ts]: I'm seeing these x's pop up from when I merged the master to update mine. I didn't add this x, nor another that I randomly found in the code. The only code I touched was anything with "console.log" in front of it. This is worrysome.

Comment (hdauvic) [reply app/misc/git.ts]: I looked at the current master and have removed this x.

File: app/components/app/app.component.ts [hunk 2]
```diff
@@ -24,7 +24,7 @@ export class AppComponent {
     // If user has previously saved a color, then set the app to that color
     if (fs.existsSync(userColorFilePath)) {
       fs.readFile(userColorFilePath, function(err, buffer) {
-        console.log(buffer.toString());
+        console.log("Last used theme: " + buffer.toString() + ", initializing to this theme.");
```
Comment (dragonprevost) [inline app/components/app/app.component.ts]: I wonder if using the console log to notify theme change is really necessary? I feel like the console is a little crowded, but that is subjective. Let me know what you think...

File: app/misc/authenticate.ts [hunk 3]
```diff
@@ -180,9 +180,10 @@ function authenticateUser(callback) {
       });
 
     }, err => {
-      console.log('Error while getting token', err);
+      console.log('ERROR while getting token. ', err);
 	}).catch( err => {
-    console.log(err);
+    let tokenError = err;
```
Comment (dragonprevost) [inline app/misc/authenticate.ts]: Is `tokenError` used other than initializing it to equal `err` ?

**Comment by dragonprevost** [comment]
Really the only thing that should maybe be changed is the tokenError variable. Great work other wise!

File: app/misc/authenticate.ts (hunk 3 above)
Comment (hdauvic) [reply app/misc/authenticate.ts]: Nope. Realized after the fact that I could just use err and it wouldn't cause any problems. Just didn't go back and fix it due to laziness.

**Pull request merged**